Source/EventAnalysis/KTOverlappingTrackClustering.cc

```diff
@@ -92,22 +134,44 @@ namespace Katydid
 
     bool KTOverlappingTrackClustering::Run()
     {
-        return DoClustering();
+        if (fCompTracks.size() != 0 )
```

Comment:
Shouldn't you decide on using one or the other depending on the type of slot you got and if the number of CompTrack is indeed non zero?